React/Fabric/Mounting/RCTComponentViewDescriptor.h

```diff
@@ -22,7 +22,7 @@ class RCTComponentViewDescriptor final {
    * Associated (and owned) native view instance.
    */
   __strong RCTUIView<RCTComponentViewProtocol> *view = nil; // [macOS]
-
+  NSInteger tag = 0; // default is 0 // [macOS]
```

Comment:
```suggestion
  NSInteger tag = 0; // [macOS] default is 0
```

Reply:
Why does macOS need this but not iOS?

Reply:
This is temp shim till we merge our TouchHandler changes with proper fix